### PR TITLE
Side bar component dev

### PR DESCRIPTION
**Pull request by arangeltorres**
### What does it do?

Describe the technical changes you did.

### Why is it needed?

Describe the issue you are solving.

### How to test it?

Below are listed every action that should be possible within the application.

#### Content Manager

**Article**

List view:

- [ ] Publication Status is working
- [ ] Review Workflows assignees & filtering is working (EE)

Entry view:

- [ ] CKEditor field is working
- [ ] Assignees and Stages are working (EE)

**Page**

Entry view:

- [ ] Creating and publishing a page is working

#### Plugins

**Content-type Builder**

- [ ] Content-type Builder is working

**Media Library**

- [ ] Media Library is working (upload)
- [ ] Folders are working

#### Settings

- [ ] Marketplace is working
- [ ] API Tokens Interface
- [ ] Internationalization Interface
- [ ] Single Sign-On Settings
- [ ] Review Workflows
  - Show custom created Workflow
  - Drag N Drop is working
  - Add new workflow stage is possible
  - Colors drop down is working
- [ ] Webhooks
  - Review Workflows Stage Change option
- [ ] Audit Logs
  - List view
  - Single log
- [ ] Roles
  - Default Roles
  - Create new role
  - Show CRUD + Publish
    - Field level
    - Locale
- [ ] Users & Permissions
  - Roles 
  - Add New Role

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request

### CODE REVIEW

**Review by gsuarez17 — APPROVED** [review]

File: src/app/components/shared/sidebar/sidebar.component.scss [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+            }
+        }
+
+        ::ng-deep .mat-expansion-panel-body{
+            padding: 0px !important;
+            display: flex;
+            flex-direction: column;
+            gap: 16px;
+            margin-top: 16px;
+            cursor: pointer;
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/sidebar/sidebar.component.scss:61]: can we add `:host ::ng-deep` in order to apply this style only in this component

File: src/app/components/shared/sidebar/sidebar.component.scss [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+        }
+
+        .business-option{
+            display: flex;
+            height: 20px;
+            align-items: center;
+            align-self: stretch;
+            .business-option-text{
+                color: var(--color-fg-secondary, #414641);
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/sidebar/sidebar.component.scss:70]: why two colors like this ? `var(--color-fg-secondary, #414641)` ain't `var(--color-fg-secondary)` does the work just the same?

File: src/app/components/shared/sidebar/sidebar.component.scss [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+        gap: 8px;
+        align-self: stretch;
+        .menu-section-title{
+            flex: 1 0 0;
+            color: var(--color-fg-tertiary, #787D78);
+            leading-trim: both;
+            text-edge: cap;
+            font-variant-numeric: lining-nums;
+
+            font-family: Inter;
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/sidebar/sidebar.component.scss:98]: same as above

File: src/app/components/shared/sidebar/sidebar.component.scss [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+                display: flex;
+                align-items: center;
+                justify-content: center;
+
+                .title-icon{
+                    font-size: 12px;
+                    display: flex;
+                    align-items: center;
+                    justify-content: center;
+                    color: var(--hc-color-bg-cta, #5A28FF);
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/sidebar/sidebar.component.scss:137]: same as above

File: src/app/components/shared/sidebar/sidebar.component.scss [hunk 5]
```diff
@@ -0,0 +1,243 @@
...
+                }
+            }
+    
+            .section-element-title{
+                color: var(--color-fg-secondary, #414641);
+                leading-trim: both;
+                text-edge: cap;
+                font-variant-numeric: lining-nums;
+    
+                font-family: Inter;
```
Comment (acamachoaltimetrik) [inline src/app/components/shared/sidebar/sidebar.component.scss:156]: same

File: src/stories/sidebar-component.stories.ts [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+import { SidebarComponent } from 'app/components/shared/sidebar/sidebar.component';
+import { BrowserAnimationsModule, provideAnimations } from '@angular/platform-browser/animations';
+
+export default {
+  title: 'Shared/Sidebar',
+  component: SidebarComponent,
+  decorators: [
+    moduleMetadata({
+      imports: [ BrowserAnimationsModule],
+      providers: [provideAnimations()]
```
Comment (acamachoaltimetrik) [inline src/stories/sidebar-component.stories.ts:11]: is this necessary here ?

**Pull request merged**